Sources/SaferContinuation/SaferContinuation.swift

```diff
@@ -51,9 +51,22 @@ final public class SaferContinuation<C: Continuation & Sendable>: @unchecked Sen
 	  - context: Allows you to provide any arbitrary data to differentiate between different continuations that you can inspect when errors are thrown or
 	 notifications posted.. Could be a string, a UUID, a UIImage, or your mom's nickname. The last one is probably not useful though. You be the judge.
 	 */
-	public init(_ continuation: C, isFatal: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions = false, timeout: TimeInterval? = nil, delayCheckInterval: TimeInterval? = 3, file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil) {
+	public init(_ continuation: C,
+                isFatal: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions? = nil,
+                timeout: TimeInterval? = nil,
+                delayCheckInterval: TimeInterval? = 3,
+                file: StaticString = #file,
+                line: Int = #line, 
+                function: StaticString = #function,
+                context: Any? = nil) {
+#if DEBUG
+        let defaultFatalOptions: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions = true
+#else
+        let defaultFatalOptions: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions = false
+#endif
+        
```

Comment:
What is the justification for changing this behavior?

I'm not saying you're wrong, I just see benefits to both approaches, I want to see if there are any real world scenarios I hadn't thought of that would benefit from this approach, especially before changing existing behavior.

Reply:
With this, we can find some potential problems during development and debugging, and there will be no fatalError at release.

Reply:
My apologies. I had the model inverted in my mind and was assuming that I previously had it default to crash on error, so I thought it was being modified to step down security, not up. This is fine :)